example/cumulus-tf/main.tf

```diff
@@ -141,6 +141,8 @@ module "cumulus" {
   additional_log_groups_to_elk  = var.additional_log_groups_to_elk
 
   tags = local.tags
+
+  enable_deprecation_warnings = var.enable_deprecation_warnings
```

Comment:
I only added a TF variable so that we could see these warnings in the task for our example Cumulus deployments, but upon further review that may not even be necessary